library/ssl_misc.h

```diff
@@ -450,6 +450,24 @@ size_t mbedtls_ssl_get_output_max_frag_len(const mbedtls_ssl_context *ssl);
 size_t mbedtls_ssl_get_input_max_frag_len(const mbedtls_ssl_context *ssl);
 #endif /* MBEDTLS_SSL_MAX_FRAGMENT_LENGTH */
 
+#if defined(MBEDTLS_SSL_RECORD_SIZE_LIMIT)
+/**
+ * \brief          Return the record size limit (in bytes) for
+ *                 the output buffer. This is less than the value requested by the
+ *                 peer (using RFC 8449), since it subtracts the space required for the
```

Comment:
```suggestion
 * \brief          Return the RecordSizeLimit  (in bytes) for
 *                 the output buffer. This is less than the value requested by the
 *                 peer (See RFC 8449), since it subtracts the space required for the
```

Reply:
Done.